src/EprPrnIntegration.Common/Configuration/RrepwApiConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+using System.Diagnostics.CodeAnalysis;
+
+namespace EprPrnIntegration.Common.Configuration;
+
+[ExcludeFromCodeCoverage]
+public class RrepwApiConfiguration
+{
+    public const string SectionName = "RrepwApi";
+    public string BaseUrl { get; set; } = null!;
```

Comment:
this overrides null checking...just make it nullable?
```suggestion
    public string? BaseUrl { get; set; } = null;
```